### DISCUSSION TITLE
Set crop ratio?

### DISCUSSION THREAD

**Discussion by vincesd8** [rsjaffe/MIDI2LR#581] [Q&A] [answered]
Is there a way to set a crop ratio? Original/4:3/3:2 etc.

**Comment by rsjaffe** [comment, marked as answer]
See [Commands: quick develop](https://github.com/rsjaffe/MIDI2LR/wiki/Commands#8), "Set crop ratio" near the bottom of that set of commands.

**Comment by vincesd8** [comment]
Awesome! Does exactly what I need. Missed it as I thought it would be under "Crop" section.
